packages/evm/contracts/budgets/ATransparentBudget.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.24;
+
+import {IERC1155Receiver} from "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
+import {IERC165} from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+
+import {ABudget} from "contracts/budgets/ABudget.sol";
+import {ACloneable} from "contracts/shared/ACloneable.sol";
+
+/// @title Abstract Simple ABudget
+/// @notice A minimal budget implementation that simply holds and distributes tokens (ERC20-like and native)
+/// @dev This type of budget supports ETH, ERC20, and ERC1155 assets only
+abstract contract ATransparentBudget is ABudget, IERC1155Receiver {
```

Comment:
We'll need to put the net-new functions in the abstract so the component interfaces work correctly

Reply:
This is still relevant at minimum we need `createBoost` and potentially `getFungibleAmountAndKey` & `getERC1155AmountAndKey`